src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/Internal/Topics.cs

```diff
@@ -44,31 +69,39 @@ internal abstract class TopicLike : ActorBase
         protected Deadline PruneDeadline = null;
 
         /// <summary>
-        /// TBD
+        /// Used to toggle what we do during publication when there are no subscribers
+        /// </summary>
+        protected readonly bool SendToDeadLettersWhenNoSubscribers;
+
+        /// <summary>
+        /// Creates a new instance of a topic or group actor.
         /// </summary>
-        /// <param name="emptyTimeToLive">TBD</param>
-        protected TopicLike(TimeSpan emptyTimeToLive)
+        /// <param name="emptyTimeToLive">The TTL for how often this actor will be removed.</param>
+        /// <param name="sendToDeadLettersWhenNone">When set to <c>true</c>, this actor will
+        /// publish a <see cref="DeadLetter"/> for each message if the total number of subscribers == 0.</param>
+        protected TopicLike(TimeSpan emptyTimeToLive, bool sendToDeadLettersWhenNone)
```

Comment:
Updated constructor to receive new value. These are all `internal` classes so no breaking changes here.